static/js/code-highlighter.js

```diff
@@ -126,8 +125,8 @@ var DocumentTitler = new (class DocumentTitler {
       return { long: bestPretty, short: bestShortPretty };
     }
 
-    const symElems = elem.querySelectorAll("[data-i]");
-    scan: for (const symElem of symElems) {
+    const defs = elem.querySelectorAll(".syn_def");
```

Comment:
Clever constraint refinement here!

I think it would probably be good to add a comment like the following since part of why this works is misleading, feel free to use or edit the comment as you desire:

format.rs maps "def", "decl", and "idl" syntax kinds to "syn_def" so this covers all "kinds" of interest.